### DISCUSSION TITLE
Axios gives no response

### DISCUSSION THREAD

**Discussion by icjcs** [quasarframework/quasar#15568] [General - Components / Directives / etc] [answered]
I try to implement an REST service. From the server I can get a answer with the browser or via curl. I adopted the REST service to signal a incoming request. This works with axios, but I get no response and no error. What do I do wrong? Here the sample code:

`<template>`
`  <q-page padding>`
`    <h3>New booking</h3>`
`    <div>{{ loadData() }} {{ data }}</div>`
`  </q-page>`
`</template>`

`<script>`
`import { ref } from "vue";`
`import axios, { api } from "boot/axios";`
`import { useQuasar } from "quasar";`

`export default {`
`  name: "newTask",`
`  setup() {`
`    const $q = useQuasar();`
`    const data = ref("undefiniert");`

`    async function loadData() {`
`      const response = await api.get("http://localhost:81/CurrentTime/get");`
`      data.value = response.data;`
`      return response.data;`
`    }`
`    return { data, loadData };`
`  },`
`};`
`</script>`

regards
CRT

**Comment by MilosPaunovic** [comment]
What does you `boot/axios` file look like?

**Reply by icjcs** [reply]
Thanks for Your interest.

// src/boot/axios.js

`import { boot } from "quasar/wrappers";`
`import axios from "axios";`

`const api = axios.create({ baseURL: "" });`

`export default boot(({ app }) => {`
`  app.config.globalProperties.$axios = axios;`
`  app.config.globalProperties.$api = api;`
`});`

`export { axios, api };`

**Comment by MilosPaunovic** [comment, marked as answer]
At first glance, everything seems fine. Try to replace the URL to your request to something you know will return response, let's say `https://jsonplaceholder.typicode.com/posts` and see if the problem is with the backend you're pointing to.

**Reply by icjcs** [reply]
Thank You, I get an answer from your suggested adress. It works. 
The problem is my server, but I wonder why. When I call by browser or curl then I get an answer. My dev tool seems to be outdated :-(
